.github/ISSUE_TEMPLATE/Feature.md

```diff
@@ -7,12 +7,19 @@ about: Suggest a new feature or enhancement for rclone
 
 Welcome :-)
 
-So you've got an idea to improve rclone? We love that! You'll be glad to hear we've incorporated hundreds of ideas from contributors already.
+So you've got an idea to improve rclone? We love that!
+You'll be glad to hear we've incorporated hundreds of ideas from contributors already.
+Probably the latest release has your feature. Try to update your rclone, run:
 
-Here is a checklist of things to do:
+    rclone self-update
```

Comment:
Should we ask to update to `beta` instead? The same conditions apply regarding the choice and revert that was mentioned for the bug template.

Reply:
Beta features can be buggy and are not guaranteed to settle before next release and the news may not be propagated across the team.
Moreover, beta features frequently exist on dedicated git branches rather than on master. A support person will have to recommend a concrete beta anyway.
This is in contrast to the bug fixing template, where *any* fix is better than *no* fix. I thought about removing beta from there too, but I was slightly biased to keep it.

Reply:
Alright, that makes sense.